ADR: Format of minigame configuration IDs (#82)
Gamify-IT/docs: adr/2022-07-17-minigame-configuration-id-format.md

# Formats minigame configuration ID

> Currently we have no clear format, in which format we create the configuration ID's. The configuration ID's generated by the minigame backends will be send to the overworld backend, so they know which configuration ID should be load when entering a specific minigame.

Reviewer: ```suggestion
Currently, we have no clear format on how to create the configuration IDs.
The configuration IDs generated by the minigame backends will be sent to the overworld backend so that they know which configuration should be loaded when entering a specific minigame.
```
